<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>image_properties_dock</class>
 <widget class="QWidget" name="image_properties_dock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Image Properties</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="rowWrapPolicy">
    <enum>QFormLayout::WrapAllRows</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblFileName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Image &amp;File:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>fileName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="KLineEdit" name="fileName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="browse">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblAspectRatio">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Positioning:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>aspectRatio</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KComboBox" name="aspectRatio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string comment="Positioning of an image in a crossword image cell.">Scaled and Cropped</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Positioning of an image in a crossword image cell.">Scaled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Positioning of an image in a crossword image cell.">Scaled (Keep Aspect Ratio)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Positioning of an image in a crossword image cell.">Centered</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Positioning of an image in a crossword image cell.">Tiled</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblHorizontalCellSpan">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Horizontal Cells:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>horizontalCellSpan</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KIntNumInput" name="horizontalCellSpan">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblVerticalCellSpan">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Vertical Cells:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>verticalCellSpan</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KIntNumInput" name="verticalCellSpan">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fileName</tabstop>
  <tabstop>browse</tabstop>
  <tabstop>aspectRatio</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
